ImgBot.Function/CompressImages.cs

```diff
@@ -124,17 +124,25 @@ public static class CompressImages
             return optimizedImages;
         }
 
-        private static string CreateCommitMessage(Dictionary<string, Percentage> optimizedImages)
+        private static string CreateCommitMessage(Dictionary<string, Tuple<double, double>> optimizedImages)
         {
             var commitMessage = new StringBuilder();
+            var totalOrigKb = 0.0;
+            var totalOptKb = 0.0;
             commitMessage.AppendLine("[ImgBot] optimizes images");
 
             foreach (var optimizedImage in optimizedImages.Keys)
             {
-                commitMessage.AppendFormat("{0} ({1})", optimizedImage, optimizedImages[optimizedImage].ToString());
+                var percent = 1 - (optimizedImages[optimizedImage].Item1 - optimizedImages[optimizedImage].Item2) * 100;
+                commitMessage.AppendFormat("{0} -- {1}kb -> {2}kb ({3})", optimizedImage, optimizedImages[optimizedImage].Item1, optimizedImages[optimizedImage].Item2, percent);
                 commitMessage.AppendLine();
+                totalOrigKb += optimizedImages[optimizedImage].Item1;
+                totalOptKb += optimizedImages[optimizedImage].Item2;
             }
 
+            commitMessage.AppendFormat("*Total: {0}kb -> {1}kb ({2})", totalOrigKb, totalOptKb, (1 - (totalOrigKb - totalOptKb) * 100));

```

Comment:
What do you think about pretending this total message to the top of the commit message instead of at the bottom?

Reply:
That makes a lot of sense -- much more discoverable.

Reply:
Cool. Also prepending not pretending. Haha

Reply:
addressing as part of https://github.com/dabutvin/ImgBot/issues/50

Thanks for the contribution @squalrus !!